openlibrary/templates/type/list/view.html

```diff
@@ -197,7 +197,7 @@
                                 $elif 'borrow_url' in ebook:
                                     <span class="actions read">
                                       <a href="$ebook['borrow_url']" class="borrow-link" title="Borrow book"
-					 data-key="$(doc.key)" data-ocaid="$(doc.get('ocaid', '') or ''.join(doc.get('ia', '')))">
+					 data-key="$(doc.key)" data-ocaid="$(doc.get('ocaid', '') or ''.join(doc.get('ia', '')) or ','.join(ebook['ia']))">
```

Comment:
I'm not sure that `''.join(doc.get('ia', ''))` does anything, unless there are editions that have 'ia' without 'ocaid'?  If this 'ia' joining was meant to be for works, it will have never worked.